Library/Homebrew/cleanup.rb

```diff
@@ -356,17 +356,6 @@ def cleanup_portable_ruby
           FileUtils.rm_rf path
         end
       end
-
-      return unless Dir.glob(portable_ruby_glob).empty?
-      return unless portable_ruby_path.exist?
-
-      bundler_path = vendor_path/"bundle/ruby"
-      if dry_run?
-        puts "Would remove: #{bundler_path} (#{bundler_path.abv})"
-        puts "Would remove: #{portable_ruby_path} (#{portable_ruby_path.abv})"
-      else
-        FileUtils.rm_rf [bundler_path, portable_ruby_path]
```

Comment:
Instead this should run `safe_system "git", "clean", "-xf", bundler_path`

Reply:
Done!